spec/system/admin/products_v3/products_spec.rb

```diff
@@ -1213,6 +1209,21 @@ def expect_other_columns_visible
             expect(input_content).to match /COPY OF Apples/
           end
         end
+
+        it "fails to clone the product on page when clicked on the cloned option" do
+          # Mock the +save+ method to return fail. That's the only expected fail case
+          allow_any_instance_of(Spree::Product).to receive(:save).and_return(false)
```

Comment:
Context for this here:
https://github.com/openfoodfoundation/openfoodnetwork/pull/12565#discussion_r1654688954